uint8 MAP_LEFT=0
uint8 MAP_LEFT_DAIZA=1
uint8 MAP_RIGHT=2
uint8 MAP_RIGHT_DAIZA=2

#map
uint8 map